version/dev/api/common.html

```diff
@@ -536,7 +536,7 @@ <h2>Connection builder<a class="headerlink" href="#connection-builder" title="Li
 </dl>
 <p class="rubric">Notes</p>
 <p>For advanced usage, including configuring session-specific properties and timeouts, see the
-<a class="reference external" href="https://openapi.docs.pyansys.com/version/stable/api/index.html" title="(in ansys.openapi.common v2.0.2)"><span class="xref std std-doc">ansys-openapi-common API reference</span></a>. Specifically, see the
```

Comment:
Is this deliberate one? I think it would be overwritten on any pushes to main

Reply:
I changed all these individual versions to 'stable', but realized that it wasn't something we'd be able to maintain, and so I changed them back. I must have missed this one, I'll revert before I merge.